sources/platform/workflow/job/overview.md

```diff
@@ -114,6 +119,59 @@ When triggered, a Job does the following:
 
 Every time a job executes, a new immutable **version** is created. This makes it easy to 'replay' older job versions, though you should do this only after verifying that the old input values will not create problems in your Assembly Line.
 
+<a name="trigger-modes"></a>
+## Job triggering modes
+
+Once a job completes successfully or a resource is updated, it looks for all the other job resources that are using it as INs and depending on the dependencyMode of those job resources, it decides whether to create a waiting or queued build for that job. Specifying dependencyMode is optional. `dependencyMode` can only be `chrono`, `immediate` or `strict`. Default dependencyMode is `chrono`.
+
+<img src="/images/platform/jobs/chrono-dependencyMode.png" alt="How are DevOps activities triggered?" style="width:100%;vertical-align: middle;display: block;margin-left: auto;margin-right: auto;"/>
+
+* **chrono dependencyMode**
+
+    This is the default dependencyMode.  When a job completes or a resource is updated, all the consistent jobs which are using this resource/job as IN are evaluated and the ones which have consistent(valid yml definition and success state) IN resources/jobs are triggered. These jobs will get processed only when all of their IN and OUT jobs are in complete state.
+
+    Example:
+
+    In the above assembly line, `job-2` and `job-3` are triggered manually. Suppose, `job-2` completes first then it will trigger `job-5` once but not `job-4` since it has an inconsistent IN `job-1`. As `job-3` is still in processing state, `job-5` will remain in queued state until `job-3` is complete. Once `job-3` completes it will trigger `job-5` again. `job-5` will have 2 queued builds for it. If `job-7` is in processing state, then `job-5` will wait for that also to complete and then both of its builds will go to processing state one by one.
+
+* **immediate dependencyMode**
+
+    As the name suggests, jobs are triggered immediately without waiting for any incomplete IN/OUT jobs to be completed and jobs also get triggered even if IN resources are inconsistent. When a job completes or a resource is updated, all the consistent jobs which are using this resource/job as IN are triggered.
```

Comment:
In this mode, jobs are triggered without waiting for any incomplete dependencies to reach a final state.  Jobs will also be triggered even if IN resources are inconsistent. When a job completes or a resource is updated, all the consistent downstream jobs are triggered.